--- /dev/null
+++ ../library/Zend/Db/Statement/Pdo.php
@@ -231,7 +231,7 @@
             }
         } catch (PDOException $e) {
             require_once 'Zend/Db/Statement/Exception.php';
-            $message = sprintf('%s, query was: %s', $e->getMessage(), $this->_stmt->queryString);
+            $message = sprintf('%s, query was: %s, params: %s', $e->getMessage(), $this->_stmt->queryString, var_export($params, true));
             throw new Zend_Db_Statement_Exception($message, (int) $e->getCode(), $e);
         }
     }
